docs/evm-sidechain/configure-the-bridge-client.md

```diff
@@ -0,0 +1,141 @@
+  - Interoperability
+status: not_enabled
+---
+# Configure the Bridge Client
+
+The client requires separate configuration files for the backend frontend. Each environment setting also requires its own configuration files.
+
+
+## Backend Configuration
+
+The client backend is configured in `src/config`. The folder holds multiple files that configure properties related to the:
+
+- Server
+- Logger
+- Database (TypeORM)
+- EVM
+- XRPL
+- XUMM
+
+These configuration files use the `buildConfig()` function, taking an object with the configuration properties and a second optional parameter to perform validations. Configuration properties can have a static value or an object specifying values for each environment. The following environments are available:
+
+- `default`
+- `production`
+- `development`
+- `test`
+- `staging`
+
+Configuration values can be constants, environment variables or AWS Secrets. Environment variables are defined in a .env file in the root of the backend. AWS Secrets add an extra layer of customisation and security. To load AWS secrets, the `AWS_REGION` and `AWS_SECRET_ID` environment variables must be defined.
+
+**Warning:** Never use constants for sensitive data, such as keys or secrets in production environments.
+
+
+### Server Options
+
+| Key | Description | EVN Key | AWS Secret | Example |
+|-----|-------------|---------|------------|---------|
+| port | Server port to receive requests. | `APP_PORT` | | `3000` |
+| secretKey | Secret key used to generate JWT tokens. | `APP_JWT_KEY` | | `"AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="` |
+| enableSwagger | Enable Swagger. | | | `true` |
+| enableCor | Enable CORS. | | | `true` |
+
+### Logger Options
+
+| Key | Description | EVN Key | AWS Secret | Example |
+|-----|-------------|---------|------------|---------|
+| logLevel | Log level used by the server logger. | | | `"info"` |
+|logFileName | File name to store logs. | | | `"app.log"` |
+
+### Database (TypeORM) Options
+
+| Key | Description | EVN Key | AWS Secret | Example |
+|-----|-------------|---------|------------|---------|
+| host | Database host. | `DB_HOST` | `DB_HOST` | `"db"` |
+| port | Database port. | `DB_PORT` | `DB_PORT` | `5432` |
+| username | Database username. | `DB_USER` | `DB_USER` | `"db_user"` |
+| password | Database password. | `DB_PASSWORD` | `DB_PASSWORD` | `"db_password"` |
+| database | Database name. | `DB_DATABASE` | `DB_DATABASE` | `"db_database"` |
+
+**Note:** This configuration also extends [Nest TypeORM properties](https://docs.nestjs.com/techniques/database).
+
+### EVM Option
+
+| Key | Description | EVN Key | AWS Secret | Example |
+|-----|-------------|---------|------------|---------|
+| server | EVM chain RPC. | | | `"https://rpc-evm-sidechain.xrpl.org"` |
+
+### XRPL Option
+
+| Key | Description | EVN Key | AWS Secret | Example |
+|-----|-------------|---------|------------|---------|
+| server | XRPL chain WebSocket. | | | `"wss://sidechain-net1.devnet.rippletest.net"` |
```

Comment:
```suggestion
| `server` | The URL of a WebSocket interface to the XRPL chain. | | | `"wss://sidechain-net1.devnet.rippletest.net"` |
```

(Also, I would expect you _might_ need to provide the port number here unless the server operator forwards port 443 to whatever port it's listening for wss connections on. Usually rippletest.net instances aren't configured that way so you have to explicitly specify a port, like `wss://s.altnet.rippletest.net:51233/`